sandbox/libs/authn/src/main/java/org/opensearch/authn/jwt/JwtVendor.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.authn.jwt;
+
+import org.apache.cxf.jaxrs.json.basic.JsonMapObjectReaderWriter;
```

Comment:
Any reason for pulling in CXF over say https://github.com/auth0/java-jwt?

Reply:
The main implementation uses [jjwt](https://github.com/jwtk/jjwt) which is also used by the security plug-in. I will have to take a deeper look at this other library to understand the trade-offs.

I will write a section in the markdown document on the libraries in use and what each is used for to understand the need for these dependencies. 

```
implementation 'io.jsonwebtoken:jjwt-api:0.10.8'
implementation('org.apache.cxf:cxf-rt-rs-security-jose:3.4.5') {
  exclude(group: 'jakarta.activation', module: 'jakarta.activation-api')
}

runtimeOnly 'org.apache.cxf:cxf-core:3.4.5'
implementation 'org.apache.cxf:cxf-rt-rs-json-basic:3.4.5'
runtimeOnly 'org.apache.cxf:cxf-rt-security:3.4.5'
```